webpack.config.extensions.js

```diff
@@ -139,6 +144,10 @@ module.exports = [
 				/^@wordpress\/i18n$/,
 				path.join( __dirname, './extensions/shared/i18n-to-php' )
 			),
+			new webpack.NormalModuleReplacementPlugin(
+				/^@wordpress\/element$/,
```

Comment:
I had one more idea.

Having `NormalModuleReplacementPlugin` replace imports from `@wordpress/element` has proven tricky, since our replacement -- [`element-to-php.js`](https://github.com/Automattic/jetpack/pull/17072/files#diff-3911155fb41ef0cc76677493b709ce3a7d1295ab5d797c72f79948d2a457e408) -- would need to include all export symbols from that package (with only [`createInterpolateElement` replaced with the identity](https://github.com/Automattic/jetpack/pull/17072/files#diff-3911155fb41ef0cc76677493b709ce3a7d1295ab5d797c72f79948d2a457e408R39)). Unlike the [counterpart](https://github.com/Automattic/jetpack/blob/a4175da6e7e2e968eff86396f20bce01be25a7e9/extensions/shared/i18n-to-php.js) for `@wordpress/i18n`, `@wordpress/element` contains a plethora of export symbols, and the mix of wildcard re-exports that we've so far tried in this PR hasn't worked; and listing them all manually would be a maintenance nightmare, and very fragile.

But the `NormalModuleReplacementPlugin` should also work at a deeper level, so we might try to replace the [`./create-interpolate-element` import at package root level](https://github.com/WordPress/gutenberg/blob/fbb000fcb495e8e3de567ae06c12a00544592c47/packages/element/src/index.js#L1) instead:

```suggestion
				/^\.\/create-interpolate-element$/,
```

(if my RegEx-fu isn't mistaken)

However, this means that we'll also need to stop externalising `@wordpress/element`, and include it in our build instead.

cc/ @brbrr @anomiex @jeherve

Reply:
Looks like that might actually work (without needing to touch the externalised stuff?) Pushed a commit to #16763 (as deps are more up-to-date there). Let's continue there.